iot_verifier/src/loader.rs

```diff
@@ -26,8 +26,8 @@ use twox_hash::XxHash64;
 use xorf::{Filter as XorFilter, Xor16};
 
 const REPORTS_META_NAME: &str = "report";
-/// cadence for how often to look for  reports from s3 buckets
-const REPORTS_POLL_TIME: u64 = 60 * 5;
+/// cadence for how often to look for  reports from s3 buckets in minutes
+const REPORTS_POLL_TIME: i64 = 5;
```

Comment:
ok.. so time for a lazy_static ChronoDuration::minutes here so you're not doing it multiple times in the code?

Reply:
all of this is changed already in another PR: https://github.com/helium/oracles/pull/312

So i wouldnt get overly hung on on those details for this change ?

Reply:
ah ok